src/channel.h

```diff
@@ -76,7 +76,7 @@ class CChannel : public QObject
 
     void PrepAndSendPacket ( CHighPrioSocket* pSocket, const CVector<uint8_t>& vecbyNPacket, const int iNPacketLen );
 
-    void ResetTimeOutCounter() { iConTimeOut = iConTimeOutStartVal; }
+    void ResetTimeOutCounter() { iConTimeOut = bDisconnectAndDisable ? 1 : iConTimeOutStartVal; }
```

Comment:
How often do we use short hand if in the source?

Reply:
> How often do we use short hand if in the source?

You mean conditional assignment ?
It is used at several places.
I use it whenever it makes the code more readable... (at least to me ;=))

Reply:
> I use it whenever it makes the code more readable

I usually think it makes code less readable ;-). But in this case I think it's ok - but that depends on if this is used elsewhere too.

Reply:
Where's it's a simple "truth" test to pick between two values based on another value, it's okay, in general, I'd say.  (I don't think Volker was keen on it.)  Anything else (e.g. an expression getting evaluated as any part of the `?:` expression).